packages/skills-gameon-sdk/src/utils/displayUtils.ts

```diff
@@ -88,6 +97,12 @@ export function renderLeaderboard(player: AugmentedPlayer,
     if (!renderOptions.secondaryColor) {
         renderOptions.secondaryColor = '#66298f';
     }
+    if (!renderOptions.scorePrimaryText) {
+        renderOptions.scorePrimaryText = `You placed ${player.score.ordinalRank}!`;
+    }
+    if (!renderOptions.scoreSecondaryText) {
+        renderOptions.scoreSecondaryText = `${player.score.score} points`;
+    }
```

Comment:
Please add a unit test that covers both of these render options and ensures that the correct APL is generated. There are some similar tests here: https://github.com/alexa-games/skills-gameon-sdk-js/blob/master/packages/skills-gameon-sdk/test/utils/displayUtils.spec.ts